theme/common.less

```diff
@@ -362,10 +415,103 @@ div.simframe > iframe {
 }
 
 /* Experimental banner */
-#experimentalBanner {
+#experimentalBanner, #windowsStoreBanner {
     z-index: 1000;
 }
 
+#windowsStoreBanner {
+    height: @bannerHeight;
+}
+
+#windowsStoreBanner img {
+    height: calc(@bannerHeight - 1rem);
+}
+
+/*******************************
+             Docs
+*******************************/
+
+#docs {
```

Comment:
Are these CSS changes related to the Windows banner? We should have a separate .less file for it.